src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs

```diff
@@ -0,0 +1,63 @@
+#nullable enable
+
+using Uno.Logging;
+using Uno.Extensions;
+using Microsoft.Extensions.Logging;
+using Windows.Data.Xml.Dom;
+using System;
+
+namespace Windows.UI.Notifications
+{
+
+	public partial class ScheduledToastNotification
+	{
+
+		public ScheduledToastNotification(XmlDocument content, DateTimeOffset deliveryTime)
+		{
+			if(content is null)
+            {
+				// yes, UWP throws here ArgumentException, and not ArgumentNullException
+				throw new ArgumentException("ScheduledToastNotification constructor: XmlDocument content cannot be null");
+            }
```

Comment:
Please use the VS formatter to adjust the indentation and spacing.

Reply:
^K D done.